.circleci/config.yml

```diff
@@ -104,7 +104,7 @@ jobs:
     steps:
       - add_ssh_keys:
           fingerprints:
-            - 'ec:9f:2e:aa:1f:c9:ab:49:57:8f:c0:cd:2e:5b:f3:b0'
+            - "ec:9f:2e:aa:1f:c9:ab:49:57:8f:c0:cd:2e:5b:f3:b0"
```

Comment:
Not sure why your prettier is reformatting this file with double quotes, can you check your settings?

Reply:
@gcpantazis Locally this didn't auto-format to single quotes for me. Setting the default formatter in vscode project settings seems to have done the trick though